src/scripts/items/ShovelItem.ts

```diff
@@ -0,0 +1,18 @@
+///<reference path="Item.ts"/>
+
+class ShovelItem extends Item {
+
+    constructor(basePrice: number, displayName: string) {
+        super('Berry_Shovel', basePrice, GameConstants.Currency.farmPoint, { multiplierDecreaser: MultiplierDecreaser.Berry }, displayName);
+    }
+
+    gain(amt: number) {
+        GameHelper.incrementObservable(App.game.farming.shovelAmt, amt);
+    }
+
+    use() {
+    }
+}
+
+// TODO: HLXII - Balance cost based on highest yield strategy
+ItemList['Berry_Shovel']   = new ShovelItem(500, 'Berry Shovel');
```

Comment:
Would be good to get all the TODOs done before merge as well where possible,
Not sure what this should cost though.
```suggestion
ItemList['Berry_Shovel']   = new ShovelItem(500, 'Berry Shovel');
```